source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/ExpropriationTabContainerView.tsx

```diff
@@ -32,9 +46,14 @@ export const ExpropriationTabContainerView: React.FunctionComponent<
     }
   };
 
+  const handleDelete = async (form8Id: number) => {
+    await deleteForm8(form8Id);
+    onForm8Deleted(form8Id);
+  };
+
   return (
     <>
-      <LoadingBackdrop show={loading} />
+      <LoadingBackdrop show={loading || deletingForm8} />
```

Comment:
moving the deletion to the container would also allow you to control the loading flag with a single prop.

Reply:
updated